draft-ietf-homenet-front-end-naming-delegation.mkd

```diff
@@ -649,25 +615,27 @@ The HNA SHOULD reject any incoming messages other than DNS NOTIFY response, SOA
 {{!RFC7368}} in Section 3.7.3 recommends DNSSEC to be deployed on both the authoritative server and the resolver.
 The resolver side is out of scope of this document, and only the authoritative part of the server is considered.
 
-This document assumes the HNA signs the Public Homenet Zone.
+It is RECOMMENDED the HNA signs the Public Homenet Zone.
 
 Secure delegation is achieved only if the DS RRset is properly set in the parent zone.
 Secure delegation is performed by the HNA or the DOIs.
 
-The DS RRset can be updated manually with nsupdate for example.
+The DS RRset can be updated manually in the parent zone with nsupdate for example.
 This requires the HNA or the DOI to be authenticated by the DNS server hosting the parent of the Public Homenet Zone.
 Such a trust channel between the HNA and the parent DNS server may be hard to maintain with HNAs, and thus may be easier to establish with the DOI.
 In fact, the Public Authoritative Server(s) may use Automating DNSSEC Delegation Trust Maintenance  {{!RFC7344}}.
 
 
 # Homenet Reverse Zone Channels Configuration {#sec-reverse}
```

Comment:
I have a suggestion for document structuring. Since in sections 5 and 6 you are discussing channels. Maybe it is better to pull channel configurations before the policy section i.e. 
6.  DM Distribution Channel . . . . . . . . . . . . . . . . . . .  19
7. HNA Channel Configurations
7.1. Homenet Public Zone
7.2 Homenet Reverse Zone    
8.  HNA Security Policies . . . . . . . . . . . . . . . . . . . .  19

I also think that the main purpose of this section is to link with the dhc-architecture document. So this section could be shortened - mainly, to me it seemed something too much detail for this document.

Reply:
I think there are different purpose, one is that we do not forget the reverse zone the other is to show it works the same way and the third that it is even easier and could be completely automated. The dhc option are basically pushing the configurations needed to completely automate the process in both cases. 

That said, it is true that given the same set of co-authors wrote these document the link is clearly there ;-).

Reply:
Maybe configurations aspects can be moved to the appendix.

Reply:
as well as Information Model for Outsourced information